### DISCUSSION TITLE
If the oap server stop, has any effect on the services?

### DISCUSSION THREAD

**Discussion by lanxiudh** [apache/skywalking#8257] [Q&A]
If the oap server stop, has any effect on the services?

**Comment by wu-sheng** [comment]
This question is too general. It is hard to say. What is effective and what is not.
You could test and measure based on your standards.

**Reply by lanxiudh** [reply]
i've tested that :  stop the oap server ,  and my services still work, it seems normal.  But i'm not sure that the agent collect data can't  transfer to oap server,  will has any effect?

**Reply by wu-sheng** [reply]
Yoi stopped server, what is the different with so-called can't transfer? Whay does can't transfer actually mean?

**Reply by lanxiudh** [reply]
yes, actually i don't know when the server stop,  the agent client how to work? it can't connect to the server, that will has some effect on my service?

**Reply by wu-sheng** [reply]
Read the source codes, that is how open sourcr works.

**Reply by lanxiudh** [reply]
Ok，thx
